### DISCUSSION TITLE
Does this library support Adafruit QT Py SAMD21 board?

### DISCUSSION THREAD

**Discussion by m-r-m-s** [khoih-prog/SAMD_TimerInterrupt#27] [Q&A] [answered]
Hello @khoih-prog, 

I have been using the Seeed Xiao SAMD21 board and its [Arduino core](https://github.com/Seeed-Studio/ArduinoCore-samd/tree/master/libraries) has TCC0 and TC3 helper libraries. The [QT Py Arduino core](https://github.com/adafruit/ArduinoCore-samd/tree/master/libraries) does not so I was looking for an alternative. Specifically I want to use your library with this BPM Clock Generator library [here](https://github.com/midilab/uClock/blob/64667b865ed2fe7a685a26cf4290e1966c70f41b/src/uClock.cpp#L33). 

Is the SAMD_TimerInterrupt library compatible with the Adafruit QT Py? Looking at your documentation it seems it is supported but I wanted to confirm since it is above my level of knowledge on these things. Thank you.

**Comment by khoih-prog** [comment, marked as answer]
HI @m-r-m-s 

Hopefully, if the board is compatible.

You have to go ahead and try to know. There are too many boards on the market, and I have no chance to have and test them all.

Good Luck,

**Reply by m-r-m-s** [reply]
> HI @m-r-m-s
> 
> Hopefully, if the board is compatible.
> 
> You have to go ahead and try to know. There are too many boards on the market, and I have no chance to have and test them all.
> 
> Good Luck,

I understand completely. Thank you for the quick response & I'll certainly let you know if they are compatible. Cheers
